README.md

```diff
@@ -0,0 +1,89 @@
+homebrew-cita
+=============
+
+## Installation
+```
+brew tap cryptape/cita
+```
+
+### Install docker
+```
+brew cask install docker
+```
+
+### Curl formula and install.
+
+```sh
+brew install ./cita.rb
+```
+
+By default, binaries installed by cita will be placed into:
+  
+```
+/usr/local/Cellar/cita/0.22
```

Comment:
What's Cellar? and please use `0.22.0`

Reply:
Okay, I'll change it. and, 
1. Cellar is a default path of `brew installed` software, it is specified by homebrew.
2. `homebrew-*` is also a convention of homebrew, [here][1] it is.

[1]: https://docs.brew.sh/Taps#repository-naming-conventions-and-assumptions

Reply:
ok. I'll fix this. I just follow the nameing style: `cita-*`

Reply:
I understand you, maybe developers will create repos named `cita-*` just like they creating `homebrew-*` soon, hhh